public/app/plugins/panel/text2/module.tsx

```diff
@@ -19,6 +19,16 @@ export const plugin = new PanelPlugin<TextOptions>(TextPanel)
         },
         defaultValue: 'markdown',
       })
+      .addColorPicker({
+        path: 'color',
+        name: 'Color',
+        description: 'just a test... no color pickers in core?',
+        settings: {
+          defaultText: 'Pick background',
+          disableNamedColors: false,
+          allowUndefined: true,
+        },
+      })
```

Comment:
This should not be commited!  but no story or core component that wants color?

Reply:
Yes we need color picker, just haven't migrated any piece of code it was necessary :)